### PR TITLE
June newsletter!

### PR DESCRIPTION
**Pull request by bethaitman**
Ready for review! Aiming to publish 4 June, if there's enough time for review.

### CODE REVIEW

File: docs/blog/newsletter-june-2019.rst [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+- If you can, consider introducing (as far as possible) a "blinded" interview process.
+- Consider training on how to be more aware of your biases.
+- Ask the questions you want the answers to.
+- Consider the subtext of questions that you do ask.
+- Exercise-based interviews can provide more relevant interactions with candidates than resumes and curated writing samples.
+
+Documentarians recommended some reading related to this topic:
+
+- `reWork Guide: Use structure and criteria <https://rework.withgoogle.com/print/guides/5443632811212800/>`__
+- `Should You Trust Your Guy in Hiring Decisions <https://www.shrm.org/ResourcesAndTools/hr-topics/talent-acquisition/Pages/Trust-Your-Gut-Hiring-Decisions.aspx>`__
```
Comment (hillaryfraley) [inline docs/blog/newsletter-june-2019.rst]: The link text should be "Gut" rather than "Guy," right?

File: docs/blog/newsletter-june-2019.rst [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+
+-------------------------------------------------------
+Order from chaos, or, a conversation about docs cleanup
+-------------------------------------------------------
+
+One generous WtD Slacker shared a blog post they'd written about getting started with organizing and curating doc sets, and the post generated a chorus of thanks and expressions of shared struggle. One notable piece of advice was to seek out and make public the often-hidden or kept-private stashes of information that individuals in a company invariably gather. Here's the blog post:
+
+https://medium.com/@brooke.wayne/how-to-organize-your-docs-bd797c616b48
+
+Over the years, this has been a really popular topic at Write the Docs conferences. If it's a problem you're interested, there's an embarassment of riches for you in the video archives:
```
Comment (hillaryfraley) [inline docs/blog/newsletter-june-2019.rst]: add "in" --
...a problem you're interested in, there's...

Comment (hillaryfraley) [reply docs/blog/newsletter-june-2019.rst]: and wow I love the phrase "embarrassment of riches" +++

File: docs/blog/newsletter-june-2019.rst (hunk 1 above)
Comment (bethaitman) [reply docs/blog/newsletter-june-2019.rst]: oh dear!

File: docs/blog/newsletter-june-2019.rst [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+- `Tackling technical debt in the docs </videos/prague/2018/tackling-technical-debt-in-the-docs-louise-fahey>`__ - Louise Fahey, Prague 2018 
+- `A year in the life of the better docs project </videos/prague/2018/a-year-in-the-life-of-the-better-docs-project-rowan-cota>`__ - Rowan Cota, Prague 2018
+
+--------------------
+DITA vs docs-as-code
+--------------------
+
+Our final discussion this month comes from a community question about what the differences are between docs-as-code and DITA. The short answer is that docs-as-code is a practice, whereas DITA is a tool and a format. So in theory, you could practise docs-as-code with DITA just as you would with markdown.
+
+However, it's not necessarily that straightforward. One issue is that if you chose to use DITA, you put a higher barrier to entry on non-tech-writer contributions: reading the XML isn't tricky, and licenses for tools can be expensive. You can help others review by publishing the content to PDF, but that may not be an ideal review format.
```
Comment (Bradamant3) [inline docs/blog/newsletter-june-2019.rst]: "reading the XML is tricky" ?? (currently "isn't")

File: docs/blog/newsletter-june-2019.rst [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+
+The toolchains also generally differ. Typically, DITA uses a closed content management system and toolchain, whereas docs-as-code matches what developers use for their code.
+
+DITA also emphasizes certain features, like content reuse, that don't come out of the box with most docs-as-code tools. However, you can often add those features in - Tom Johnson's `blog series comparing DITA and Jekyll <https://idratherbewriting.com/2015/03/23/new-series-jekyll-versus-dita/>`__ discusses this. He warns that with lots of customisations, you risk locking yourself into a particular toolchain - but others pointed out that it's never "easy" to change docs systems anyway.
+
+Some people pointed out the ideological differences. A lot of the point of DITA is the structure, allowing sophisticated content re-use. Docs-as-code/markdown/SSGs don’t come with the same feature set because they sometimes no have structure at all, even to the point of being anti-structure.
+
+So while you can potentially unite the two, it's worth remembering that the philosophies underlying docs-as-code and DITA are quite different.
+
```
Comment (Bradamant3) [inline docs/blog/newsletter-june-2019.rst:104]: Beth, this is a brilliant writeup of a difficult conversation. We all knew you rocked before this, but now you rock specially in this case!

**Comment by Bradamant3** [comment]
Caught one small thing -- overall wow this is a brilliant newsletter. We are so lucky to have @bethaitman!

**Review by clairealyssa — APPROVED** [review]
I didn't see anything glaring. Good job, Beth! :)

File: docs/blog/newsletter-june-2019.rst (hunk 3 above)
Comment (bethaitman) [reply docs/blog/newsletter-june-2019.rst]: Oops!

File: docs/blog/newsletter-june-2019.rst (hunk 4 above)
Comment (bethaitman) [reply docs/blog/newsletter-june-2019.rst:104]: Aw, thank you so much!

**Comment by bethaitman** [comment]
Aw, thanks everyone! @plaindocs - do you know if that last job posting is going to come in today?

**Comment by bethaitman** [comment]
@partyparrot18

**Comment by clairealyssa** [comment]
Merging per Slack request from Beth

**Pull request merged**